### PR TITLE
TRT-932: Automated alerts data update

### PR DESCRIPTION
**Pull request by deepsm007**
This PR will start updating alerts data along with disruption data every week
/cc @dgoodwin @xueqzhan

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -2741,6 +2741,14 @@ periodics:
         --leeway $leeway_percent \
         --google-service-account-credential-file /etc/bigquery/token
 
+        echo "Updating alerts data"
+        /usr/bin/job-run-aggregator analyze-historical-data \
+        --current ./pkg/monitortestlibrary/allowedalerts/query_results.json \
+        --output-file ./pkg/monitortestlibrary/allowedalerts/query_results.json \
+        --data-type alertss \
```
Comment (dgoodwin) [inline ci-operator/jobs/infra-periodics.yaml:2748]: typo

Comment (deepsm007) [reply ci-operator/jobs/infra-periodics.yaml]: fixed it, nice catch

**Comment by dgoodwin** [comment]
Gotta request testplatform approval.

**Pull request merged**